internal/dataplane/parser/atc/field.go

```diff
@@ -53,18 +66,29 @@ func (f IntField) String() string {
 	return string(f)
 }
 
+func NewIntField(value int) IntField {
```

Comment:
IntField is a field that has a integer value, not an integer name. So I think there is no need to have a `NewIntField` to create a `IntField` from an integer.

Reply:
This and the StringField one aren't too useful given that they just do type conversions, but I felt weird having it for some and not others.

Reply:
Removed.